src/main/java/net/pms/configuration/RendererConfiguration.java

```diff
@@ -1829,23 +1829,29 @@ public boolean isDefaultVBVSize() {
 	/**
 	 * Returns the maximum bitrate (in megabits-per-second) supported by the
 	 * media renderer as defined in the renderer configuration. The default
-	 * value is "0" (unlimited).
+	 * value is 0 (unlimited).
 	 *
 	 * @return The bitrate.
 	 */
-	// TODO this should return an integer and the units should be bits-per-second
-	public String getMaxVideoBitrate() {
+	public int getMaxVideoBitrate() {
 		if (PMS.getConfiguration().isAutomaticMaximumBitrate()) {
 			try {
-				return calculatedSpeed();
+				int calculatedSpeed = calculatedSpeed();
+				if (calculatedSpeed >= 70) { // this should be a wired connection
+					setAutomaticVideoQuality("Automatic (Wired)");
```

Comment:
If we are changing this setting automatically now, we should also remove it from the first-run wizard, as it would be a waste of the user's time to set it

Reply:
Also maybe we should disable that input box with a note that says something like "Video quality is handled automatically when automatic maximum bandwidth is enabled"